File: L3/java 3/Pentagon.java
public class Pentagon extends Figure {
    private int side;

    Pentagon(int side) {
        this.side = side;
    }

    public double area() {
        return side*side*Math.sqrt(25+10*Math.sqrt(5))/4;
    }

    public double perimeter() {
        return 5*side;
    }


}

File: L3/cpp 3/src/pentagon.cpp
#include <pentagon.hpp>
#include <cmath>


        Pentagon::Pentagon(int side) {
            this->side = side;
        }

        double Pentagon::area() {
            return side*side*sqrt(25+10*sqrt(5))/4;
        }

        double Pentagon::perimeter() {
            return 5*side;
        }
